docpages/example_programs/using_coroutines.md

```diff
@@ -1,7 +1,5 @@
 \page using-coroutines Using Coroutines
 
-\include{doc} coro_warn.dox
```

Comment:
The `coro_warn.dox` needs to stay. People needs to know that Coro is only supported on certain compiler versions. Just remove the last part of the `coro_warn.dox` :)